integr8ly/integreatly-operator
# [MGDAPI-3890] bump 3scale-operator to 0.8.3+0.1649688682.p

# Issue link
https://issues.redhat.com/browse/MGDAPI-3890

# What
Add 3scale-operator manifest for version `0.8.3+0.1649688682.p`

# Verification steps
- Install RHOAM via olm from this index image: `quay.io/tdimov0/managed-api-service-index:1.21.0`
- Once installed, setup workload web app on your cluster (https://github.com/integr8ly/workload-web-app#rhoam-clusters)
- Update catalog source to point to a new index: `quay.io/tdimov0/managed-api-service-index:1.21.1`
- Approve the upgrade
- Wait for the upgrade to complete and check the total downtime in 3scale (should not be longer than 2mins)
- Verify rhoam's version is `1.21.1` and 3scale-operator's version is `0.8.3`
